lib/ally_steps.js

```diff
@@ -0,0 +1,18 @@
+const { accessibility } = require('taiko');
+const assert = require("assert");
+const MIN_ACCESSIBILITY_SCORE = process.env.min_accessibility_score || 80;
+
+step("Run a11y check", async () => {
+  const audit = await accessibility.runAudit();
+
+  gauge.message(
+    `Accessibility Score: ${audit.score} Violations: ${JSON.stringify(
+      audit.violations,
+      null,
+      5
+    )}`
+  );
+  if (process.env.stop_on_a11y_fail === "true") {
```

Comment:
For backward compatibility, set the default value of this property to fail. This will otherwise break existing behavior.  Also update this in README to highlight the option.

Reply:
Added an alias in a more user friendly language